lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -345,19 +346,44 @@ def estimate_diffusion_ols(self, max_lag=None):
 
         1) Michalet, X., & Berglund, A. J. (2012). Optimal diffusion coefficient estimation in
         single-particle tracking. Physical Review E, 85(6), 061916.
+        2) Bullerjahn, J. T., von Bülow, S., & Hummer, G. (2020). Optimal estimates of
+        self-diffusion coefficients from molecular dynamics simulations. The Journal of Chemical
+        Physics, 153(2), 024116.
 
         Parameters
         ----------
         max_lag : int (optional)
-            Number of lags to include. When omitted, the method uses an optimal number of lags
-            as determined by [1].
+            Number of lags to include. When omitted, the method will choose an appropriate number
+            of lags to use.
```

Comment:
kinda makes it sound like `pylake` is sentient 😆 maybe `the method determines an appropriate number of lags` or `the method calculates an appropriate number of lags`, I'm not sure